src/coreclr/debug/ee/functioninfo.cpp

```diff
@@ -2326,12 +2333,13 @@ void DebuggerMethodInfoTable::DeleteEntryDMI(DebuggerMethodInfoEntry *entry)
 
 #endif // #ifndef DACCESS_COMPILE
 
-DebuggerJitInfo *DebuggerJitInfo::GetJitInfoByAddress(const BYTE *pbAddr )
+DebuggerJitInfo *DebuggerJitInfo::GetJitInfoByAddress(const BYTE *pbAddr)
```

Comment:
I think you deleted the only call to this method in the codebase, you can delete this method now.